### PR TITLE
#166272797 Implement babel and nodemon setup

### PR DESCRIPTION
**Pull request by adex001**
#### What does this PR do?
- Implement babel and nodemon setup
#### Description of Task to be completed?
- install nodemon as a dev dependency
- install babel-node, babel-cli, babel-core, babel-preset-env, babel-register, morgan as a dependency
- create a .babelrc file and configure babel-preset-env
- create a start and start:dev script in the package.json
- ignore package-lock.json in .gitignore file
#### How should this be manually tested?
- Clone the repository, npm i to install all dependencies and npm start, then open Postman and navigate the endpoint.
#### Any background context you want to provide?
-  N/A
#### What are the relevant pivotal tracker stories?
[#166272797](https://www.pivotaltracker.com/story/show/166272797)
#### Screenshots (if appropriate)
- N/A
#### Questions:
- N/A

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -34,3 +34,4 @@ node_modules
 # Optional REPL history
 .node_repl_history
 
+package-lock.json
```
Comment (fidelisojeah) [inline .gitignore]: Remove this

File: .babelrc [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
+  "presets": ["@babel/preset-env"]
+}
```
Comment (fidelisojeah) [inline .babelrc:16]: `@babel/plugin-proposal-object-rest-spread`
`@babel/plugin-transform-arrow-functions`

the `@babel/preset-env` should target node

**Review by fidelisojeah — CHANGES_REQUESTED** [review]
Also consider updating some files: specifically `index.js` to use more es6+ like features like `import/export` and anonymous  functions

File: .gitignore (hunk 1 above)
Comment (adex001) [reply .gitignore]: Resolved this.

File: .babelrc (hunk 2 above)
Comment (adex001) [reply .babelrc:16]: I have added the plugins and made sure @babel/preset-env targets node

File: config/index.js [hunk 3]
```diff
@@ -1,4 +1,4 @@
-module.exports = {
+export default {
```
Comment (fidelisojeah) [inline config/index.js]: perhaps:
```javascript
export const secret = process.env.NODE_ENV === 'production' ? process.env.SECRET : 'secret'
```
instead ?

**Review by fidelisojeah — CHANGES_REQUESTED** [review]
Fix linting issues

**Review by fidelisojeah — APPROVED** [review]

**Pull request merged**